pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -130,18 +135,19 @@ func TestAcc_ScimIntegration_complete(t *testing.T) {
 					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "enabled", "false"),
 					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "scim_client", "GENERIC"),
 					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "run_as_role", role.Name),
-					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "network_policy", networkPolicy.Name),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "network_policy", sdk.NewAccountObjectIdentifier(networkPolicy.Name).FullyQualifiedName()),
 					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "sync_password", "false"),
 					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "comment", "foo"),
 					resource.TestCheckResourceAttrSet("snowflake_scim_integration.test", "created_on"),
 				),
 			},
 			{
-				ConfigDirectory:   acc.ConfigurationDirectory("TestAcc_ScimIntegration/complete"),
-				ConfigVariables:   m(),
-				ResourceName:      "snowflake_scim_integration.test",
-				ImportState:       true,
-				ImportStateVerify: true,
+				ConfigDirectory:         acc.ConfigurationDirectory("TestAcc_ScimIntegration/complete"),
+				ConfigVariables:         m(),
+				ResourceName:            "snowflake_scim_integration.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"name"},
```

Comment:
Okay, so this is a downside of not setting the name. So maybe we should just set it? What were the downsides again?

@sfc-gh-jmichalak @sfc-gh-jcieslak

Reply:
I answered in the thread above that maybe we should set it, so it's read during the import, but I'm not sure about exact upsides/downsides of reading it in Read operation.

Reply:
Reverted setting name in Read. Removed ImportStateVerifyIgnore.